common/src/main/java/com/discordsrv/common/config/main/MainConfig.java

```diff
@@ -47,7 +47,7 @@ public abstract class MainConfig implements Config {
     public static final String HEADER = String.join("\n", Arrays.asList(
             "Welcome to the DiscordSRV configuration file",
             "",
-            "Looking for the \"BotToken\" option? It has been moved into the %1",
+            "Looking for the \"BotToken\" option? It has been moved into %1",
```

Comment:
This seems weird, 
"It has been moved into the connections.yaml"
"It has been moved into connections.yaml"

Reply:
"It has been moved into the connections.yaml file" would make sense as well. Without saying file it just feels a little redundant though